web/beacon-app/src/features/teams/components/DeleteMember/DeleteMemberModal.tsx

```diff
@@ -0,0 +1,78 @@
+import { Modal } from '@rotational/beacon-core';
+import { useEffect } from 'react';
+import { toast } from 'react-hot-toast';
+
+import { Close } from '@/components/icons/close';
+import Button from '@/components/ui/Button/Button';
+import { useDeleteMember } from '@/features/members/hooks/useDeleteMember';
+
+import DeleteMemberForm from './DeleteMemberForm';
+
+type DeleteMemberModalProps = {
+  onOpen: {
+    opened: boolean;
+    member?: any;
+  };
+  onClose: () => void;
+};
+
+function DeleteMemberModal({ onClose, onOpen }: DeleteMemberModalProps) {
+  const { opened, member } = onOpen;
+  const { deleteMember, isDeletingMember, wasMemberDeleted, hasMemberFailed, error, reset } =
+    useDeleteMember(member?.id);
+  const initialValues = {
+    id: member?.id,
+    name: member?.name || member?.email || '---',
+    delete_agreement: false,
+  };
+
+  const onDelete = () => {
+    deleteMember();
+  };
+
+  useEffect(() => {
+    if (wasMemberDeleted) {
+      toast.success('Success! You have removed your teammate from your organization');
+      onClose();
+      reset();
+    }
+  }, [wasMemberDeleted, onClose, reset]);
+
+  useEffect(() => {
+    if (hasMemberFailed) {
+      toast.error(
+        (error as any)?.response?.data?.error ||
+          `Could not delete member. Please try again or contact support,  if the problem continues.`
```

Comment:
```suggestion
          `We could not complete the action. Please try again.`
```

Reply:
Another message from the Figma file. If we change to this message, the error message in `AddNewMemberModal` should also be updated.

If the message doesn't change, should the space before `if the problem continues.` be deleted?